srcpkgs/png++-devel/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'png++-devel'
+pkgname=png++-devel
```

Comment:
should be png++

Reply:
So no -devel package, headers in the main package